decidim-core/app/cells/decidim/redesigned_follow_button/show.erb

```diff
@@ -10,12 +10,14 @@
     <%= render :content %>
   <% end %>
 <% else %>
-  <%= action_authorized_link_to(
+  <%= action_authorized_button_to(
```

Comment:
I have some weird behavior regarding the follow button that I captured from the redesign app (also happening locally), see the below screencap:

https://user-images.githubusercontent.com/864340/207843082-2c2a84db-6f47-4564-b90e-4fa1dc8448a0.mp4

In that situation I was signed in to the application.

This seems to happen only on the first time when arriving to the site (with cache cleared) and any further clicks to that follow button work fine.

Reply:
@ahukkanen good catch, it seems that the first load sometimes don't consider the `redesign_active` flag, while the following ones do. So the first times calls the old cell `follow_button` while the next time calls `redesigned_follow_button`.

It might take a lot of time to investigate, seems to me a caching issue of Rails instance vars, and it shouldn't be a problem once the redesign is enabled by default and the old cell doesn't exist.

Do you mind to continue with the review?

Reply:
OK, I moved it to #9671.